src/components/Dashboard/LegalObligation.vue

```diff
@@ -1,43 +1,33 @@
     class="legal-obligation-container"
   >
     <v-card flat class="legal-obligation-section">
-      <v-icon color="blue darken-2" class="info-icon">mdi-information-outline</v-icon>
-      <div class="share-structure-check-text">
-        <span class="font-weight-bold">Legal Obligations:</span>
-        You are required by the <em>Business Corporations Act</em> to keep the information about your corporation up
-        to date with the Registrar: For example, you must file annual reports, director changes and address changes.
-        <span class="app-blue cursor-pointer" @click="readMoreFlag = true">
-          <span v-if="!readMoreFlag">Read more about your legal obligations...</span>
+      <v-icon color="primary" class="mr-2">mdi-information-outline</v-icon>
+      <div>
+        <span class="font-weight-bold obligations-title">{{getObligations.title}}</span>
+        <span class="obligations-copy">
+          You are required by the <em>{{getObligations.act}}</em> {{getObligations.obligationStatement}}
         </span>
+        <p class="app-blue cursor-pointer mt-2 mb-0" @click="readMoreFlag = true">
+          <span v-if="!readMoreFlag"><u>Read more about your obligations</u></span>
+        </p>
 
         <div v-if="readMoreFlag">
-          <div class="read-more-line">
-            The most common updates you are legally responsible to file include:
-          </div>
-          <div class="read-more-line">
+          <div class="obligations-copy pt-2">{{getObligations.subtitle}}</div>
+          <div class="pt-2">
             <ul>
-              <li>
-                <span class="font-weight-bold">Annual Reports</span> - file an annual report each year within two
-                months after each anniversary of the date on which the company was recognized. This ensures your
-                company remains in good standing with the Registrar.
-              </li>
-              <li>
-                <span class="font-weight-bold">Director changes</span> - update director information within 15 days of
-                any change (appointing, ceasing or updating an existing director's name or address)
-              </li>
-              <li>
-                <span class="font-weight-bold">Company address changes</span> - update any changes to a company's
-                registered or records office addresses as they occur.
+              <li v-for="(changes, index) in getObligations.includedChanges" :key="index" class="obligations-copy">
+                <span class="font-weight-bold">{{changes.label}}</span> - {{changes.description}}
               </li>
             </ul>
           </div>
-          <div class="read-more-line">
-            <span>Find more detailed information <a :href="detailInfoURL" target="_blank">here</a></span>
+          <div class="pt-2">
+            <a :href="getObligations.detailInfoURL" target="_blank">Find out more detailed information</a>
```

Comment:
For some reason sonar is saying this is a security issue? Any thoughts on this?

Binding a computer property to a url prop is not new, so i'm not sure 🤷

Reply:
Maybe it's not happy about the lack of type for `detailInfoURL` ?

I don't think there's a security risk here. Are you able to login to Sonarcloud to change this hotspot to "Safe" status? If not, I can do it.

Reply:
Interesting, i will check it out!

Reply:
Added an interface 🤞

Reply:
Looks like it still needs the Sonar Approval, i'm not sure i can update this.
Logged in, however it still said i required permissions.

Reply:
OK, I updated this:

![image](https://user-images.githubusercontent.com/10002889/153964668-82e1cfc3-9d23-4487-8c4b-72207f069618.png)

Reply:
Thank you!